pjp.tm10/src/main/webapp/FormattingWithCSS/css/ALL.css

```diff
@@ -0,0 +1,124 @@
+@charset "ISO-8859-1";
+#Q1body
+{
+	background-image: url("../img/img1.png");
+	background-repeat: no-repeat;
+	background-position: right top;	
+}
+
+
+#Q2body
+{
+	background-color: #5ef7c2;
+	background-image: url("../img/img1.png");
+	background-repeat: no-repeat;
+	background-position: right top;	
+}
+
+
+#Q3body
+{
+	background-color: #5ef7c2;	

```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5784138)